src/heretic/r_main.c

```diff
@@ -879,7 +879,9 @@ void R_SetupFrame(player_t * player)
     if (player->fixedcolormap)
     {
         fixedcolormap = colormaps + player->fixedcolormap
-            * 256 * sizeof(lighttable_t);
+            // [crispy] sizeof(lighttable_t) not needed in paletted render
```

Comment:
Just get away with it, not worth a comment.

Reply:
Fixed in https://github.com/fabiangreffrath/crispy-doom/pull/1111/commits/1f815528656b5c297fd8fe7d9c5a4c333bba0912. But should I remove `/* * sizeof(lighttable_t)*/` as well? A bit scarry for possible diff in future, like in Chocolate Doom `sizeof` exist, but here it doesn't. The reason might be not obivious for someone who haven't suffered this issue.